### ISSUE TITLE
Could you provide testing code?

### ISSUE THREAD

**Issue by rabintang** [alexander-rakhlin/CNN-for-Sentence-Classification-in-Keras#6]
there is only training code in the repository, could you offer testing code?

**Comment by alexander-rakhlin** [comment]
It prints validation accuracy during training. If you need to predict, add
`y = model.predict(x, batch_size=32, verbose=0)`

**Comment by rabintang** [comment]
My main issue is how to vectorize testing texts and how to handle the out of vocabulary words?
